core/templates/components/state-editor/state-editor.directive.ts

```diff
@@ -121,7 +121,19 @@ angular.module('oppia').directive('stateEditor', [
             $scope.conceptCardIsShown = !$scope.conceptCardIsShown;
           };
 
+          var questionSuggestionDiv = angular.element(
+            document.getElementById('questionSuggestionDiv'));
+          questionSuggestionDiv.on('scroll', function() {
+            if (questionSuggestionDiv.scrollTop() > 50) {
```

Comment:
How was 50 chosen? Can you create a constant variable for this?

Reply:
Done.